source/resource/qx/tool/schema/compile-1-0-0.json

```diff
@@ -269,6 +260,10 @@
             "description": "If true, the source file will be saved in the map file if the target supports it. Can be overridden on a per application basis.",
             "type": "boolean"
           },
+          "source-map-relative-paths": {
+            "description": "If true, the source map files will be saved with relative paths in the map file if the target supports it. Can be overridden on a per application basis.",
```

Comment:
is it the source map file or the source file ?

Reply:
It's the path to the source file. 

{
  "version":3,
  "sources":[
    "../../usr/lib/node_modules/@qooxdoo/compiler/node_modules/@qooxdoo/framework/source/class/qx/Bootstrap.js",
   ...
}

Reply:
@hkollmann the two descriptions do not align!